apps/vgknextgame/vgk_next_game.star

```diff
@@ -0,0 +1,59 @@
+"""
+Applet: VGK Next Game
+Summary: Shows next VGK Game
+Description: Shows the date of the next Vegas Golden Knights game.
+Author: theimpossibleleap
+"""
+
+load("encoding/base64.star", "base64")
+load("http.star", "http")
+load("render.star", "render")
+load("time.star", "time")
+
+timestamp = time.now().format("2006-01-02")
+
+vgkNextGameWeek = "https://api-web.nhle.com/v1/club-schedule/VGK/week/" + timestamp
+
+def main():
+    response = http.get(vgkNextGameWeek.format(ttl_seconds = 3600))
+
+    if response.status_code != 200:
+        fail("Server request failed with status %d", response.status_code)
+
+    if len(response.json()["games"]) == 0:
```

Comment:
Not a big issue at all, but from a performance perspective it would typically be good to only call `response.json()` once. So e.g.:
```
d = response.json()
if len(d["games"]) == 0:
 ...
```